apps/wallet-mobile/src/TxHistory/TxHistoryNavigator.tsx

```diff
@@ -190,11 +191,42 @@ export const TxHistoryNavigator = () => {
 
                     <Stack.Screen
                       name="exchange-create-order"
-                      component={CreateExchangeOrder}
                       options={{
-                        title: strings.rampOnOffTitle,
+                        title: strings.exchangeCreateOrderTitle,
                       }}
-                    />
+                    >
+                      {() => (
+                        <Boundary>
+                          <CreateExchangeOrder />
+                        </Boundary>
+                      )}
+                    </Stack.Screen>
+
+                    <Stack.Screen
+                      name="exchange-select-buy-provider"
+                      options={{
+                        title: strings.exchangeSelectBuyProvider,
+                      }}
+                    >
+                      {() => (
+                        <Boundary>
```

Comment:
I think this is all static now, maybe let to wrap when it gets async from the package